### DISCUSSION TITLE
Being Verified

### DISCUSSION THREAD

**Discussion by raymate** [revoxhere/duino-coin#955] [Q&A] [answered]
When I look at my current stats on the https://explorer.duinocoin.com/ page.

Under my Balance I see Verified and my status is No.

How do you get verified. Look at few other people mining and they are verified, what is this status for / meaning ?

**Comment by gsrbr** [comment, marked as answer]
Hello @raymate  for your account be verified see this [link](https://server.duinocoin.com/verify.html) and follow the steps

**Comment by primitt** [comment]
https://server.duinocoin.com/verify.html

Follow the steps here ^
